### PR TITLE
Custom map path + removed unused import.

### PR DESCRIPTION
**Pull request by benjaminrwilson**
Updated custom map path and removed an unused import in `map_api.py`

### CODE REVIEW

File: argoverse/map_representation/map_api.py [hunk 1]
```diff
@@ -54,8 +50,12 @@ class ArgoverseMap:
     are not provided, but can be hallucinated if one considers an average lane width.
     """
 
-    def __init__(self) -> None:
+    ROOT = Path(__file__).resolve().parents[2] / "map_files"
```
Comment (johnwlambert) [inline argoverse/map_representation/map_api.py:53]: i prefer
```
ROOT = Path(__file__).resolve().parent.parent.parent  # ../../..
```

File: argoverse/evaluation/detection/utils.py [hunk 2]
```diff
@@ -93,6 +94,7 @@ class DetectionCfg(NamedTuple):
     tp_normalization_terms: np.ndarray = np.array([tp_thresh, MAX_SCALE_ERROR, MAX_YAW_ERROR])
     summary_default_vals: np.ndarray = np.array([MIN_AP, tp_thresh, MAX_NORMALIZED_ASE, MAX_NORMALIZED_AOE, MIN_CDS])
     eval_only_roi_instances: bool = True
+    map_root: Union[str, Path] = Path(__file__).parents[3] / "map_files"
```
Comment (johnwlambert) [inline argoverse/evaluation/detection/utils.py:97]: inline comment explaining "argoverse/ "evaluation "detection" / "
parent.parent.parent

File: argoverse/map_representation/map_api.py (hunk 1 above)
Comment (johnwlambert) [reply argoverse/map_representation/map_api.py:53]:
```
_PathLike = Union[str, "os.PathLike[str]"]
```
https://docs.python.org/3/library/os.html#os.PathLike

**Comment by johnwlambert** [comment]
Thanks @benjaminrwilson, it looks good. But CI is still failing because of the 3.6. So does Tox no longer support 3.6? I'm fine removing 3.6 if we add 3.8 instead

**Comment by benjaminrwilson** [comment]
Hmm, py36 doesn't appear to be deprecated from the project page: https://github.com/tox-dev/tox. Maybe related to this: [](https://github.com/tox-dev/tox/issues/1975)?

File: argoverse/map_representation/map_api.py [hunk 3]
```diff
@@ -85,6 +89,12 @@ def __init__(self) -> None:
             self.city_to_lane_bboxes_dict[city_name] = lane_bboxes
             self.city_to_da_bboxes_dict[city_name] = da_bboxes
 
+    @property
+    def map_files_root(self) -> Path:
+        if self.root is None:
+            raise ValueError("Root cannot be None!")
```
Comment (johnwlambert) [inline argoverse/map_representation/map_api.py:95]: maybe "Map root directory cannot be None!" is clearer?

**Review by johnwlambert — APPROVED** [review]

File: argoverse/map_representation/map_api.py (hunk 3 above)
Comment (benjaminrwilson) [reply argoverse/map_representation/map_api.py:95]: 👍

File: argoverse/map_representation/map_api.py [hunk 4]
```diff
@@ -54,8 +53,13 @@ class ArgoverseMap:
     are not provided, but can be hallucinated if one considers an average lane width.
     """
 
-    def __init__(self) -> None:
+    # argoverse-api/map_files
+    ROOT = Path(__file__).resolve().parent.parent.parent / "map_files"
```
[the remark this answers is not in the document]
Comment (benjaminrwilson) [reply argoverse/map_representation/map_api.py]: I went ahead and moved it up near line 44.

**Review by johnwlambert — APPROVED** [review]

**Comment by johnwlambert** [comment]
Thanks @benjaminrwilson

**Pull request merged**